docs/install/tutorials/rpi.rst

```diff
@@ -218,6 +228,8 @@ Finally, start the access point system service ``hostapd`` and the DHCP and DNS
 
 .. code-block:: console
 
+  sudo systemctl unmask hostapd
+  sudo systemctl enable hostapd
   sudo systemctl start hostapd
   sudo systemctl start dnsmasq
```

Comment:
After following the instructions to this point, hostapd was for some reason masked for me. These steps unmasked and allowed it to be enabled/started.

Reply:
Thanks, will add in #118

Reply:
Confirming that it's masked on Buster. Since `unmask` and enable can be run harmlessly several times, it doesn't matter if it was different on Stretch. Thanks! :)